### PR TITLE
Search votes by ticket token feature

### PR DESCRIPTION
**Pull request by thi4go**
Implements feature requested on #803.

Currently, UI opens a modal for proposals with active or finished voting, so the user can search by his ticket's token address, and confirm if his vote was issued properly.

### CODE REVIEW

File: src/components/Modal/contents/SearchVotesModal.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import modalConnector from "../../../connectors/modal";
+import proposalVoteResults from "../../../connectors/proposalVoteResults";
+import DcrdataTxLink from "../../DcrdataTxLink";
+import { Field, reduxForm } from "redux-form";
+import Message from "../../Message";
+
+class SearchVotesModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props.onFetchProposalVoteResults(this.props.me.payload.id);
```
Comment (brunobraga95) [inline src/components/Modal/contents/SearchVotesModal.js]: side effects should go in the componentDidMount

File: src/components/Modal/contents/SearchVotesModal.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            onSubmit={handleSubmit(this.handleSubmit)}
+          >
+            {isLoading ? (
+              <i
+                className="fa fa-circle-o-notch fa-spin right-margin-5"
+                style={{ fontSize: "26px" }}
+              />
+            ) : (
+              <div className="user-search-results">
+                <div>
```
Comment (brunobraga95) [inline src/components/Modal/contents/SearchVotesModal.js]: do you need this div?

**Comment by thi4go** [comment]
Hey @brunobraga95 thanks for the review! 

Duly noted, and applied all the suggestions to the code. Think it's good to go now. 👍

**Comment by thi4go** [comment]
Hey @fernandoabolafio thanks for the review and UI suggestions! It's much better now. Commited and rebased, so good to go in. 👍

**Comment by thi4go** [comment]
Hey @tiagoalvesdulce thanks for the review and the defensive programming tips ! Did the changes and commited 👍

**Pull request merged**